### PR TITLE
Latest commit 5e09074d587114ef4d771fe65c911b67735d4fcb by @17Halbe br…

### PR DESCRIPTION
**Pull request by srslynow**
Access control features are nice, but they break the default setup.. These files are required for startup. Were removed in previous commit 5e09074d587114ef4d771fe65c911b67735d4fcb

### CODE REVIEW

**Comment by 17Halbe** [comment]
Those files shouldn‘t be neccessary anymore. And be fully optional. 
It‘s being tested if they exist during mailserver startup and if so, the required entries in main.cf are made. 
Do you still experience startup problems?

**Comment by 17Halbe** [comment]
Should actually work as expected, otherwise the tests wouldn‘t have been past successfully.

**Comment by srslynow** [comment]
Try sending an email, you'll get a sender adress rejected error. Startup works fine because those files are only checked at receive or send events.

**Comment by 17Halbe** [comment]
Thanks for the report. I might have overlooked something. It’s working fine here...
So merge please. It won‘t hurt anyway.


...von unnerwegs...

**Comment by 17Halbe** [comment]
Hmm. Not here.. I‘ll check again. This isn‘t as elegant as I intended it to be, ;) but it‘s a definitly working workaround. 
And I‘m still quite a bit puzzled why this isn‘t working.

**Comment by 17Halbe** [comment]
Ah, got the same problem... BUT the docker image has not been updated yet. It doesn't reflect the changes I implemented.
Can you have a look if you got a result of the following command:
`docker exec mail cat /usr/local/bin/start-mailserver.sh |grep access_control`
replace mail of course with your container name..

**Comment by 17Halbe** [comment]
@johansmitsnl Are you also in charge of the docker builds? Can you get them running?

File: config/postfix-receive-access.cf [hunk 1]
```diff
@@ -0,0 +1 @@
+-
```
Comment (17Halbe) [inline config/postfix-receive-access.cf]: missing new line

**Comment by johansmitsnl** [comment]
@17Halbe yes I can restart them.

**Comment by srslynow** [comment]
@17Halbe Seems you were right about the update, ```docker exec mail cat /usr/local/bin/start-mailserver.sh |grep access_control``` doesn't return anything. The errors were definitely pointing towards non-existant ```/tmp/docker-mailserver/postfix-receive-access.cf``` and ```/tmp/docker-mailserver/postfix-send-access.cf``` files though.

**Comment by 17Halbe** [comment]
Yes, and you are still right, since the problem's not solved yet! Your PR wouldn't have helped though either! ;)  
Really appreciated your PR, otherwise we wouldn't have noticed docker isn't building the image anymore! :+1:

**Comment by srslynow** [comment]
Strange, creating these files did fix the problem for me. I have the extra newlines in those files on my linux box running the mailserver. The MR was created from a Windows laptop, and I erroneously left out the required newlines there. Anyways, you're aware of the issue, thanks for the fast responses!

**Comment by boaorm** [comment]
I guess this happened to me too when rebuilding everything this morning. So now I'm a bit worried of using v2tec/watchtower for auto-updating this project. My intention is to use docker-mailserver in 'production' (personal use).

**Comment by srslynow** [comment]
@boaorm this is the exact setup I'm running as well, I'm going to exclude this image from watchtower today unfortunately. This is not the first time things break. I love this image and I'm really grateful to the contributors but its releases are just not stable enough for blind auto-updating.

**Pull request closed without merging**